pr_insight/tools/pr_code_suggestions.py

```
@@ -311,12 +312,13 @@
             self.git_provider.publish_comment(pr_comment)
 
     def extract_link(self, s):
-        r = re.compile(r"<!--.*?-->")
-        match = r.search(s)
+        from bs4 import BeautifulSoup, Comment
```

Ruff

Redefinition of unused `BeautifulSoup` from line 7

(F811)